processing_chain/src/test/groovy/org/orbisgis/processingchain/ProcessingChainTest.groovy

```diff
@@ -292,7 +294,7 @@ class ProcessingChainTest {
                 "sugar cane":["produce":["sugar_cane"],"crop":["sugar_cane"]]
         ]
 
-         H2GIS h2GIS = H2GIS.open("./target/osm_processchain")
+         H2GIS h2GIS = H2GIS.open("./target/osm_processchain", "sa", "")
```

Comment:
Why not use the default one

Reply:
the default is null and it does not work anymore on my computer... And I can not manage to modify these values in the database

Reply:
Seems a bug in OrbisDATA @SPalominos

Reply:
l'll take a look on it

Reply:
On my side it works well. Did you try after deleting the target folder or with a `mvn clean install` ?

Reply:
yes it works for all cases except the one create by GPetit, which has "sa" as id and "" as password...

Reply:
ok